omnigibson/systems/micro_particle_system.py

```diff
@@ -1564,6 +1564,7 @@ def cm_create_particle_template(cls):
         fixed_base=False,
         visual_only=True,
         include_default_states=False,
+        abilities={},
```

Comment:
nit: when `include_default_states=False`, why do we still need to pass in `abilities={}` explicitly?